tools/fregrid/conserve_interp_util.c

```diff
@@ -154,3 +155,139 @@ void malloc_xgrid_arrays( int nsize, int **i_in, int **j_in, int **i_out, int **
   }
 
 }
+
+/*******************************************************************************
+  void get_CellStruct
+  populate CellStruct
+*******************************************************************************/
+void get_CellStruct(const int tile_in, const int nx_in, const int nxgrid, int *i_in, int *j_in,
```

Comment:
@mlee03 
Can you please add comments to new functions.

Reply:
Added